File: programacao/Ex16.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[]){
  float l01, l02, l03;
  
  printf("\n Digite o tamanho do lado 01: ");
  scanf("%f", &l01);
 
  printf("Digite o tamanho do lado 02: ");
  scanf("%f", &l02);
 
  printf("Digite o tamanho do lado 03: ");
  scanf("%f", &l03);
 
  if ( ((l01 + l02) > l03) &&
       ((l01 + l03) > l02) &&
       ((l02 + l03) > l01) )
       {
           if ( (l01 == l02) && (l01 == l03) )
           {
               printf("Triangulo Equilatero!");
           }
           else if( (l01 != l02) && (l01 != l03) )
           {
               printf("Triangulo Escaleno!");
           }
           else
           {
               printf("Triangulo Isosceles!");
           }
       }
       else
       {
           printf("Nao e Triangulo!");
       }
 
  return 0;
}

File: python/Ex16.py
lado1 = float(input("Digite o tamanho do lado1: "))
lado2 = float(input("Digite o tamanho do lado2: "))
lado3 = float(input("Digite o tamanho do lado3: "))
if(lado1 < lado2):
    temp = lado1
    lado1 = lado2
    lado2 = temp
if(lado2 < lado3):
    temp = lado2
    lado2 = lado3
    lado3 = temp
if(lado1 < lado2):
    temp = lado1
    lado1 = lado2
    lado2 = temp
print(f"lado1 : {lado1: .2f}")
print(f"lado2 : {lado2: .2f}")
print(f"lado3 : {lado3: .2f}")
if(lado1 < (lado2 + lado3)):
    if(lado1 == lado2):
        if(lado1 == lado3):
            print("Triangulo é equilatero")
        else:
            print("Triangulo é isosceles")
    else:
        if(lado2 == lado3):
            print("Triangulo é isosceles")
        else:
            print("Triangulo é escaleno")
else:
    print("Os lados não formam um triangulo")